### ISSUE TITLE
Add USGS Stations

### ISSUE THREAD

**Issue by kknee** [glos/myglos#13]
This is done, but wanted to note the following from @lukecampbell:

They're back. There's one known issue that will probably persist until we can figure out how to deal with the scale of data. We normally harvest from 3rd party sources every hour, but USGS has over 10k stations and we don't have the computational resources to harvest that frequently at the moment so on the map you will usually see them transparent and some aggregations will have NaNs in the latest timestep until the harvest is complete which takes HOURS. 

In summary:
- USGS Buoys will appear transparent when added to the map unless a harvest just completed
- While a harvest is being performed, the latest value for a particular parameter may be indicated as
